.github/workflows/build-and-deploy-on-version.yml

```diff
@@ -0,0 +1,60 @@
+name: Build and deploy on test
```

Comment:
Change this to "Build and deploy" since the two jobs work on both 'test' and 'prod'